<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/PedraPapelTesoura (2).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblImagem">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>165</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../icons/sun.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPedra">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>457</y>
     <width>110</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0%)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPCPoints">
   <property name="geometry">
    <rect>
     <x>927</x>
     <y>600</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225,205,105)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerPoints">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225,205,105)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPapel">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>386</y>
     <width>110</width>
     <height>137</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0%)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>532</y>
     <width>110</width>
     <height>138</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,80%);
color:rgb(225,205,150);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTesoura">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>457</y>
     <width>110</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0%)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>181</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBright">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>530</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/pngwing.com.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelection">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/pngwing.com.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>532</y>
     <width>110</width>
     <height>138</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,50,20,50%)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus_2">
   <property name="geometry">
    <rect>
     <x>759</x>
     <y>132</y>
     <width>290</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(205,50,200,50%);
border-radius:145px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRounds">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>185</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225,205,105)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRounds_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225,205,105)</string>
   </property>
   <property name="text">
    <string>Round</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 2px solid rgb(52, 59, 72);
border-radius: 35px; 
background-color: rgb(52, 59, 72);
color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblBackground</zorder>
  <zorder>btnReset_2</zorder>
  <zorder>lblStatus_2</zorder>
  <zorder>lblImagem</zorder>
  <zorder>btnPedra</zorder>
  <zorder>lblPCPoints</zorder>
  <zorder>lblPlayerPoints</zorder>
  <zorder>btnPapel</zorder>
  <zorder>btnStart</zorder>
  <zorder>btnTesoura</zorder>
  <zorder>btnReset</zorder>
  <zorder>lblBright</zorder>
  <zorder>lblSelection</zorder>
  <zorder>lblStatus</zorder>
  <zorder>lblRounds</zorder>
  <zorder>lblRounds_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
